tests/testthat/test-replace_missing_values.R

```diff
@@ -0,0 +1,36 @@
+data <- readRDS(system.file("extdata", "test_df.RDS", package = "cleanepi"))
+test_that("replace_missing_values works", {
+  cleaned_data <- replace_missing_values(
+    data           = data,
+    target_columns = "sex",
+    na_strings     = "-99"
+  )
+  expect_s3_class(cleaned_data, "data.frame")
+  expect_false("-99" %in% cleaned_data[["sex"]])
+  expect_true(anyNA(cleaned_data[["sex"]]))
+})
+
+test_that("replace_missing_values works", {
+  cleaned_data <- replace_missing_values(
+    data           = data,
+    target_columns = NULL,
+    na_strings     = "-99"
+  )
+  expect_s3_class(cleaned_data, "data.frame")
+  expect_false("-99" %in% cleaned_data[["sex"]])
+  expect_false("-99" %in% cleaned_data[["dateOfBirth"]])
+  expect_true(anyNA(cleaned_data[["sex"]]))
+  expect_true(anyNA(cleaned_data[["dateOfBirth"]]))
```

Comment:
This could be brittle in cases where the original data had both `NA` and `-99` as NA.

Reply:
make sense - will replace this with your recommendation above to make sure it is checking on where the input data was "-99".